packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -303,7 +309,7 @@ export class StripeWebhookHandler extends StripeHandler {
       );
       return;
     }
-    this.stripeHelper.refreshCachedCustomer(uid, account.email);
+    this.stripeHelper.removeCustomerFromCache(uid, account.email);
```

Comment:
What is this about? Are there other places where we are still pulling a customer record from Redis? Will this cause those places to break?

Reply:
When processing these events, the data is not used again right here, and in fact, might not be used for hours/days afterwards. But refresh forces a new copy of the customer record into Redis, even though it might not be used, why? So instead this merely removes the record from Redis as we know there's a been a change invalidating it, and if its actually needed later, a new fetch will be done on-demand.